src/Entity/EntityId.php

```diff
@@ -17,6 +18,30 @@
 
 	protected $serialization;
 
+	const PATTERN = '/^:?(\w+:)*[^:]+\z/';
```

Comment:
Why should :a:Q1 fail? It should just be normalized to "a:Q1". 
To wit, ":a:Q1" is valid as input, but should always be normalized. No EntityId would ever return a serialization starting with : from getSerialization.